docs/user-guide/kubectl-conventions.md

```diff
@@ -38,7 +38,8 @@ In order for `kubectl run` to satisfy infrastructure as code:
 * Deployment, using `extensions/v1beta1` endpoint - use `deployment/v1beta1` (default).
 * Deployment, using `apps/v1beta1` endpoint - use `deployment/apps.v1beta1` (recommended).
 * Job - use `job/v1`.
-* CronJob - use `cronjob/v2alpha1`.
+* CronJob - using `batch/v1beta1` endpoint - use `cronjob/v1beta1`(default).
+* CronJob - using `batch/v2alpha1` endpoint - use `cronjob/v2alpha1` (deprecated).
```

Comment:
Remove this from docs since it's deprecated (and we don't encourage people to use it)?

Reply:
I've always seen this file as an information. We had this for extensions.Jobs with that same information. Especially to give people time to migrate from one to the other, when they do use it. There are not that many places where we can announce deprecation, this is a good place, imho. I'd suggest leaving this as is.